c3.php

```diff
@@ -170,6 +171,14 @@ function __c3_build_crap4j_report(PHP_CodeCoverage $codeCoverage, $path)
         return $path . '.crap4j.xml';
     }
 
+	function __c3_build_cobertura_report(PHP_CodeCoverage $codeCoverage, $path)
```

Comment:
This function is defined, but not used,
add `cobertura` route to switch statement: https://github.com/Codeception/c3/blob/8f432510c9b0e7293c6ec1fe5d56e261cba1a018/c3.php#L315-L351

Reply:
I added cobertura to the switch statement.